tests/test_efficientnet.py

```diff
@@ -0,0 +1,285 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+from typing import TYPE_CHECKING
+from unittest import skipUnless
+
+import torch
+from parameterized import parameterized
+
+from monai.networks import eval_mode
+from monai.networks.nets import EfficientNetBN, drop_connect, get_efficientnet_image_size
+from monai.utils import optional_import
+from tests.utils import skip_if_quick, test_pretrained_networks, test_script_save
+
+if TYPE_CHECKING:
+    import torchvision
+
+    has_torchvision = True
+else:
+    torchvision, has_torchvision = optional_import("torchvision")
+
+if TYPE_CHECKING:
+    import PIL
+
+    has_pil = True
+else:
+    PIL, has_pil = optional_import("PIL")
+
+
+def get_model_names():
+    return ["efficientnet-b{}".format(d) for d in range(8)]
+
+
+def get_expected_model_shape(model_name):
```

Comment:
Could you please also add test cases for different input shapes?

Reply:
What do you mean by this? The input shapes vary with each version of the network. All tests have been implemented with respective input shapes for the networks. See https://github.com/Project-MONAI/MONAI/blob/d83a053f1c14ea4f90787bef1be4263e00076d5c/tests/test_efficientnet.py#L79 where this is done.

Reply:
I mean for actual situation, the network should also work for other shapes, rather than only the original shapes, like [192, 192].

Reply:
Okay, I will setup some tests for it. Do you have any suggestions of nice shapes to cover for medical imaging related problems?

Reply:
> I mean for actual situation, the network should also work for other shapes, rather than only the original shapes, like [192, 192].

This has been done now for shapes in [128, 256, 512] - all of which are non-default to the networks:

https://github.com/Project-MONAI/MONAI/blob/45c2d91b1554354dbc1ccbd1016ba407554a2d64/tests/test_efficientnet.py#L204